player_app/serializers.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework.serializers import ModelSerializer
+from .models import Player, PlayerTraits, GameProgress
+
+class PlayerSerializer(ModelSerializer):
+    class Meta:
+        model = Player
+        fields = ['id', 'name', 'user']
+
+
+class PlayerTraitsSerializer(ModelSerializer):
+    class Meta:
+        model = PlayerTraits
+        fields = ['id', 'charisma', 'dexterity', 'strength', 'intelligence', 'player_id']
```

Comment:
whenever you are grabbing all fields you could do 
`fields = ["__all__"]`

it may save you some time